addon/lib/shooter.js

```diff
@@ -104,6 +105,8 @@ function extractWorker(tab, timeLimit) {
     });
     watchWorker(worker);
     worker.port.on("data", function (data) {
+      let timeFinished = new Date();
+      sendTiming("addon", "make-shot", timeFinished - timeStarted);
```

Comment:
It's the pokiest stage.  The `collectInformation` method in shooter.js is a good place to catch this info.  I don't know how GA does visualizing different stages, but the add-ids stage (that should generally be very fast, and blocks the rest), then extractWorker (which is slow), the make-static-html framescript call (which is usually pretty fast, but is worth watching), there's the full screen screenshot (which I think is fast but might be worth capturing in case it sometimes isn't).

Also in _activateWorker there's the handler for the "select" message, and that does the clip images.  Those might also be worth measuring...?